cmd/cofidectl/cmd/trustzone/trustzone.go

```diff
@@ -164,6 +166,7 @@ func (c *TrustZoneCommand) GetAddCommand() *cobra.Command {
 	f.StringVar(&opts.trustDomain, "trust-domain", "", "Trust domain to use for this trust zone")
 	f.StringVar(&opts.kubernetesCluster, "kubernetes-cluster", "", "Kubernetes cluster associated with this trust zone")
 	f.StringVar(&opts.context, "kubernetes-context", "", "Kubernetes context to use for this trust zone")
+	f.StringVar(&opts.trustProvider, "trust-provider", "kubernetes", "Cofide trust provider used in the installation (e.g. kubernetes, unix)")
 	f.StringVar(&opts.profile, "profile", "kubernetes", "Cofide profile used in the installation (e.g. kubernetes, istio)")
```

Comment:
Unsure about the relationship between these two. I was assuming we would only expose the profile for now, and set the trust provider kind based on the profile. Probably we should at least set the trust provider default based on the profile.